src/components/chat/comment-card.test.tsx

```diff
@@ -7,6 +7,7 @@ import { UserModelType } from "../../models/stores/user";
 import { CommentCard } from "./comment-card";
 
 jest.mock("../../hooks/use-stores", () => ({
+  useTypeOfTileInDocumentOrCurriculum: () => "Text",
```

Comment:
The mock that I slacked to you last week is more flexible in that it allows testing alternate conditions, but in the interest of moving this along let's not worry about it right now.
```typescript
var mockUseTypeOfTileInDocumentOrCurriculum = jest.fn();
jest.mock("../../hooks/use-stores", () => ({
  useTypeOfTileInDocumentOrCurriculum: mockUseTypeOfTileInDocumentOrCurriculum
});

it("should render the document icon when no tile is selected", () => {
  mockUseTypeOfTileInDocumentOrCurriculum.mockImplementation(() => undefined);
  ...rest of test code...
});
it("should render the text tool icon when a text tile is selected", () => {
  mockUseTypeOfTileInDocumentOrCurriculum.mockImplementation(() => "Text");
  ...rest of test code...
});
```